### PR TITLE
Add d[bB]/d[eE]/y[bB]/y[eE] bindings

### PR DESCRIPTION
**Pull request by mxkrsv**

### CODE REVIEW

**Comment by hamrik** [comment]
It looks like you've accidentally left the b/B motions bound to w/W.

File: config.toml [hunk 1]
```diff
@@ -72,6 +72,10 @@ up = ["select_mode", "extend_to_line_bounds", "extend_line_above", "yank_main_se
 G = ["select_mode", "extend_to_line_bounds", "goto_last_line", "extend_to_line_bounds", "yank_main_selection_to_clipboard", "delete_selection", "normal_mode"]
 w = ["move_next_word_start", "yank_main_selection_to_clipboard", "delete_selection"]
 W = ["move_next_long_word_start", "yank_main_selection_to_clipboard", "delete_selection"]
+w = ["move_prev_word_start", "yank_main_selection_to_clipboard", "delete_selection"]
+W = ["move_prev_long_word_start", "yank_main_selection_to_clipboard", "delete_selection"]
```
Comment (shawnohare) [inline config.toml]:
```suggestion
b  = ["move_prev_word_start", "yank_main_selection_to_clipboard", "delete_selection"]
B = ["move_prev_long_word_start", "yank_main_selection_to_clipboard", "delete_selection"]
```

File: config.toml [hunk 2]
```diff
@@ -83,6 +87,10 @@ up = ["select_mode", "extend_to_line_bounds", "extend_line_above", "yank_main_se
 G = ["select_mode", "extend_to_line_bounds", "goto_last_line", "extend_to_line_bounds", "yank_main_selection_to_clipboard", "collapse_selection", "normal_mode"]
 w = ["move_next_word_start", "yank_main_selection_to_clipboard", "collapse_selection", "normal_mode"]
 W = ["move_next_long_word_start", "yank_main_selection_to_clipboard", "collapse_selection", "normal_mode"]
+w = ["move_prev_word_start", "yank_main_selection_to_clipboard", "collapse_selection", "normal_mode"]
+W = ["move_prev_long_word_start", "yank_main_selection_to_clipboard", "collapse_selection", "normal_mode"]
```
Comment (shawnohare) [inline config.toml]:
```suggestion
b = ["move_prev_word_start", "yank_main_selection_to_clipboard", "collapse_selection", "normal_mode"]
B  = ["move_prev_long_word_start", "yank_main_selection_to_clipboard", "collapse_selection", "normal_mode"]
```

Comment (mxkrsv) [reply config.toml]: Indeed, idk why such a mistake :skull: :skull: :skull:

**Pull request closed without merging**